examples/basic/cli/src/index.ts

```diff
@@ -173,6 +173,32 @@ async function runTests() {
 
     console.log(intersectResultbbox.data.map(data => data.usState));
 
+    const testAggregateCountNoGroup = await client.objects.BoundariesUsState
+      .aggregateOrThrow({
+        select: { $count: true, latitude: ["min", "max", "avg"] },
+      });
+
+    // Should be 51 because it includes DC
+    console.log(
+      testAggregateCountNoGroup.$count,
+      testAggregateCountNoGroup.latitude.avg,
+      testAggregateCountNoGroup.latitude.max,
+      testAggregateCountNoGroup.latitude.min,
+    );
+    const testAggregateCountWithGroups = await client.objects.BoundariesUsState
+      .aggregateOrThrow({
+        select: { $count: true, latitude: ["min", "max", "avg"] },
```

Comment:
Ok so the types are working, but there's one weird thing happening. When I do something like:
`const testAggregateCountWithGroups = await client.objects.BoundariesUsState
      .aggregateOrThrow({
        select: { $count: true, misNameLatitiude: ["min", "max", "avg"] },
        groupBy: { usState: "exact" },
      });` the type checks let it through and it doesn't red line(the results are still fine though and don't pass through misNameLatitude)
     ```
 But if I do something like:
      const testAggregateCountWithGroups = await client.objects.BoundariesUsState
      .aggregateOrThrow({
        select: { $count: true, misNameLatitiude: ["min", "max", "avg"] },
        groupBy: { misNameUsState: "exact" },
      }); 
```
_then_ it recognizes that both the select and groupBy have a misnamed property. The groupby also correctly typechecks on its own.
**TLDR: the select statement only complains about misnamed or nonexistent param keys in its type if theres also a type issue with one of the other agg clauses like groupBy **